soc/arm/microchip_mec/mec172x/reg/mec172x_gpio.h

```diff
@@ -575,7 +615,7 @@ struct gpio_ctrl2_regs {
 	volatile uint32_t CTRL2_0036;
 	uint32_t RSVD3[1];
 	volatile uint32_t CTRL2_0040;
-	uint32_t RSVD4[1];
+	volatile uint32_t CTRL2_0041;
```

Comment:
Any specific reason for the exclusion of below member?

CTRL2_0001
CTRL2_0005
CTRL2_0006
CTRL2_0230 to CTRL2_0236
CTRL2_0251 and CTRL2_0252